src/main/java/com/arangodb/springframework/core/convert/ArangoEntityWriter.java

```diff
@@ -20,22 +20,15 @@
 
 package com.arangodb.springframework.core.convert;
 
-import org.springframework.data.convert.EntityWriter;
 
-import com.arangodb.velocypack.VPackBuilder;
-import com.arangodb.velocypack.VPackSlice;
+import com.fasterxml.jackson.databind.JsonNode;
 
 /**
  * @author Mark Vollmary
  * @author Christian Lechner
  *
  */
-public interface ArangoEntityWriter extends EntityWriter<Object, VPackBuilder> {
-
-	default VPackSlice write(final Object source) {
-		final VPackBuilder builder = new VPackBuilder();
-		write(source, builder);
-		return builder.slice();
-	}
+public interface ArangoEntityWriter {
```

Comment:
why no longer implement `EntityWriter`?

Reply:
EntityWriter is an `Interface to write objects into store specific sinks`. But here we have no need to append data invoking `write()` mutiple time, instead the method can directly return the final `JsonNode`.
IMHO the use we made of it before was unneeded:
```
public interface ArangoEntityWriter extends EntityWriter<Object, VPackBuilder> {

	default VPackSlice write(final Object source) {
		final VPackBuilder builder = new VPackBuilder();
		write(source, builder);		// <-- here is the only place where EntityWriter.write() is invoked
		return builder.slice();
	}

}
```